docs/guides/specify-runtime-options.mdx

```diff
@@ -0,0 +1,384 @@
+
+#### Options class
+
+When creating an instance of the `Estimator` or `Sampler` class, you can pass in an instance of the options class. Those options will then be applied when you use `run()` to perform the calculation.  Specify the options in this format:  `options.option.sub-option.sub-sub-option = choice`.  For example: `options.dynamical_decoupling.enable = True`
+
+Example:
+
+<Tabs>
+  <TabItem value="PrimV2" label="V2 primitives">
+    `SamplerV2` and `EstimatorV2` have separate options classes ([`EstimatorOptions`](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.EstimatorOptions) and [`SamplerOptions`](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.SamplerOptions)).
+
+```python
+from qiskit_ibm_runtime import QiskitRuntimeService
+from qiskit_ibm_runtime import EstimatorV2 as Estimator
+from qiskit_ibm_runtime.options import EstimatorOptions
+
+service = QiskitRuntimeService()
+backend = service.least_busy(operational=True, simulator=False)
+
+options =  EstimatorOptions(resilience_level=2, resilience={"zne_mitigation": True, "zne": {"noise_factors": [1, 3, 5]}})
+
+# or...
+options =  EstimatorOptions()
+options.resilience_level = 2
+options.resilience.zne_mitigation = True
+options.resilience.zne.noise_factors = [1, 3, 5]
+
+estimator = Estimator(mode=backend, options=options)
+```
+  </TabItem>
+
+  <TabItem value="PrimV1" label="V1 primitives">
+```python
+from qiskit_ibm_runtime import Estimator, Options
+
+options = Options(resilience_level = 2)
+
+# or...
+
+options = Options()
+options.resilience_level = 2
+
+estimator = Estimator(session=backend, options=options)
+```
+  </TabItem>
+</Tabs>
+
+#### Dictionary
+
+You can specify options as a dictionary when initializing the primitive.
+
+<Tabs>
+  <TabItem value="PrimV2" label="V2 primitives">
+ ```python
+from qiskit_ibm_runtime import QiskitRuntimeService
+from qiskit_ibm_runtime import EstimatorV2 as Estimator
+
+service = QiskitRuntimeService()
+backend = service.least_busy(operational=True, simulator=False)
+
+# Setting options during primitive initialization
+estimator = Estimator(backend, options={"resilience_level": 2, "resilience": {"zne_mitigation": True, "zne": {"noise_factors": [1, 3, 5]}}})
+```
+  </TabItem>
+
+  <TabItem value="PrimV1" label="V1 primitives">
+ ```python
+from qiskit_ibm_runtime import QiskitRuntimeService
+from qiskit_ibm_runtime import Estimator
+
+service = QiskitRuntimeService()
+backend = service.least_busy(operational=True, simulator=False)
+
+# Setting options during primitive initialization
+estimator = Estimator(backend, options={"resilience_level": 2})
+```
+  </TabItem>
+</Tabs>
+
+### Update options after initialization
+
+You can specify the options in this format: `primitive.options.option.sub-option.sub-sub-option = choice` to take advantage of auto complete, or use the `update()` method to make bulk updates.
```

Comment:
```suggestion
You can specify the options in this format: `primitive.options.option.sub-option.sub-sub-option = choice` to take advantage of auto-complete, or use the `update()` method to make bulk updates.
```
(or make it one word? I can't find definitively which is better, hyphenated or one word)

Reply:
I think hyphenated.